### PR TITLE
Fix CR_ACIDDEMONSTRATION proccing strip status on mobs

### PR DESCRIPTION
**Pull request by Tydus1**
* **Addressed Issue(s)**: #5906
* **Server Mode**: Pre-Renewal
* **Description of Pull Request**: Added a check to prevent skill_break_equip function to be called when Acid Demonstration and AM_ACIDTERROR are used against mobs.
* This solves the adressed issue. However, analyzing the code, skills like Tarot Card of Fate (Gypsy/Clown), Head Crush (Lord Knight), some 3rd job skills... have the same effect of breaking enemy equipments and they all call skill_break_equip so... maybe all those skills should NOT inflict strip status when used against mobs? As I don't have all this information, i'm just pull requesting a fix for Acid Demonstration & Acid Terror (I assume they have the same behavior...)

### CODE REVIEW

**Review by aleos89 — APPROVED** [review]

File: src/map/skill.cpp [hunk 1]
```diff
@@ -1628,6 +1629,9 @@ int skill_additional_effect(struct block_list* src, struct block_list *bl, uint1
 		break;
 
 	case CR_ACIDDEMONSTRATION:
+		if (!dstmd)	{
```
Comment (RagnaWay) [inline src/map/skill.cpp:1632]: wouldn't it be better to remove this TAB and replace it with a SPACE

Comment (aleos89) [inline src/map/skill.cpp:1632]:
```suggestion
		if (!dstmd) {
```

**Review by aleos89 — APPROVED** [review]

**Comment by Lemongrass3110** [comment]
This is mixed between pre-renewal and renewal right now.
Is it intended for both or just one of the modes?

**Comment by Tydus1** [comment]
Fixed in hash:https://github.com/rathena/rathena/commit/038c1778e18aa52dfd62d57a49d06f1055422e04

**Pull request closed without merging**